openassessment/xblock/staff_area_mixin.py

```diff
@@ -226,7 +235,83 @@ def render_student_info(self, data, suffix=''):  # pylint: disable=W0613
+            # If no submissions are available for graidng, will reutrn None.
+            submission_to_assess = staff_api.get_submission_to_assess(course_id, item_id, staff_id)
+
+            if submission_to_assess is not None:
+                submission = submission_api.get_submission_and_student(submission_to_assess['uuid'])
+                if submission:
+                    anonymous_student_id = submission['student_item']['student_id']
+                    submission_context = self.get_student_submission_context(
+                        self.get_username(anonymous_student_id), submission
+                    )
+                    path = 'openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html'
+                    return self.render_assessment(path, submission_context)
+                else:
+                    return self.render_error(self._(u"Error loading the checked out learner response."))
+            else:
+                return self.render_error(self._(u"No other learner responses are available for grading at this time."))
+
+        except PeerAssessmentInternalError:
+            return self.render_error(self._(u"Error getting staff grade information."))
+
+    def get_student_submission_context(self, student_username, submission):
+        """
+        Get a context dict for rendering a student submission and associated rubric (for staff grading).
+        Includes submission (populating submitted file information if relevant), rubric_criteria,
+        and student_username.
+
+        Args:
+            student_username (unicode): The username of the student to report.
+            submission (object): A submission, as returned by the submission_api.
+
+        Returns:
+            A context dict for rendering a student submission and associated rubric (for staff grading).
+        """
+        if submission and 'file_key' in submission.get('answer', {}):
+            file_key = submission['answer']['file_key']
+
+            try:
+                submission['file_url'] = file_api.get_download_url(file_key)
+            except file_exceptions.FileUploadError:
+                # Log the error, but do not prevent the rest of the student info
+                # from being displayed.
+                msg = (
+                    u"Could not retrieve image URL for staff debug page.  "
+                    u"The learner username is '{student_username}', and the file key is {file_key}"
+                ).format(student_username=student_username, file_key=file_key)
+                logger.exception(msg)
+
+        context = {
+            'submission': create_submission_dict(submission, self.prompts) if submission else None,
+            'rubric_criteria': copy.deepcopy(self.rubric_criteria_with_labels),
+            'student_username': student_username,
+        }
+
+        if self.rubric_feedback_prompt is not None:
```

Comment:
@andy-armstrong this and below passes the rubric_feeback_prompt and default text so we can delete the hard-coded text in the templates (and get what staff has actually set in Studio).